green_mood_tracker/Roberta-pretrained.py

```diff
@@ -0,0 +1,142 @@
+import tensorflow as tf
+import pandas as pd
+import string
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+from nltk import download
+import twint
+import pandas as pd
+import os
+import matplotlib.pyplot as plt
+import seaborn as sns
+from sklearn.model_selection import train_test_split
+from green_mood_tracker.training_data import get_raw_data
+from green_mood_tracker.data_cleaning import clean
+import tensorflow_datasets as tfds
+from transformers import TFRobertaForSequenceClassification
+from transformers import RobertaTokenizer
+from tensorflow.keras.callbacks import EarlyStopping
+from tensorflow.keras import models
+
+max_length = 30
+batch_size = 256
+max_length = 30
+learning_rate = 7e-5
+epsilon = 1e-8
+number_of_epochs = 10
+patience = 5
+sample=True
+roberta_tokenizer = RobertaTokenizer.from_pretrained("roberta-base")
+
+def data_prep(sample=True):
+    raw_data = get_raw_data()
+    raw_data_clean = clean(raw_data,'text')
+    raw_data_clean = raw_data_clean[raw_data_clean['polarity']!=1]
+    data_sample = raw_data_clean[raw_data_clean['source']!='sentiment140']
```

Comment:
the get_raw_data function takes a binary argument, that would do the steps, we can refactor after, so no need to change now :)